internal/sync/sync.go

```diff
@@ -71,12 +71,17 @@ func NewSynchronizer(cfg *config.Sync, feederClient *feeder.Client, syncManager
 	synchro := new(Synchronizer)
 	synchro.logger = Logger.Named("Sync Service")
 	synchro.feeder = feederClient
-	if !cfg.Trusted {
+	trusted := cfg.EthNode == ""
```

Comment:
Let's add spaces here for clarity because `trusted` is also used on line 96.

```suggestion

trusted := cfg.EthNode == ""

```